operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -576,9 +562,15 @@ private void buildIngressController(String name, String domain,
         createOrEdit(builder.build(), existing);
     }
 
+    // for testing
+    void setAzReplicaCount(Optional<Integer> value) {
+        azReplicaCount = value;
+    }
+
     int numReplicasForZone(LongSummaryStatistics ingress, LongSummaryStatistics egress,
             long connectionDemand, double zonePercentage) {
         // use the override if present
+        int desiredReplicaCount = nodeInformer.getList().size() > 0 ? 1:0;
```

Comment:
Could this be called `minimumReplicaCount` ?

Reply:
corrected.